src/databricks/labs/ucx/source_code/jobs.py

```diff
@@ -441,3 +493,85 @@ def _process_dependency(
         linter = FileLinter(ctx, path_lookup, self._session_state, dependency.path, inherited_tree)
         for advice in linter.lint():
             yield LocatedAdvice(advice, dependency.path)
+
+
+def _get_path_modified_datetime(path: Path) -> datetime:
+    unix_time = 0.0
```

Comment:
All branches effectively just set a single value… I'm not sure why the current initialize/add branching control flow is in place?

Reply:
Fixed